cmd/up/main.go

```diff
@@ -216,7 +152,40 @@ func main() {
 	level.Info(l).Log("msg", "up completed its mission!")
 }
 
-func addCustomQueryRunGroup(ctx context.Context, g *run.Group, l log.Logger, opts options, m metrics, cancel func()) {
+func write(ctx context.Context, l log.Logger, opts options.Options) error {
```

Comment:
it would be super cool if we had one interface that both packages could satisfy so we wouldn't have to care here. esp if we eventually want to add other signals. nbd and we can do in a follow up :)

Reply:
I would say we should keep the interface work for later, when we can properly vendor loki and use its protobufs. WDYT?

Reply:
yes definitely nor required in this pr